### PR TITLE
Database path with enviorment variabels

### PR DESCRIPTION
**Pull request by HenrijetteOest**
**### 1/2 pull request** 
Azure has a enviorment variabel that our program gobally hosts our data. Therefor we have an other local enviorment variabel. for our local run of database.

**Primary changes to Program**:
Other than enviorment variabel changes, 
Updated connectionstring, locally and gobally information given to strings
Now all of it runs in a main method as well

**Ubuntu latest update (other pull request)**
In 3xworkflow files I changed "ubuntu-latest" to "ubuntu-latest-22.04" because the ubuntu has an upcomming update. 
(As seen in some warnings in Build and deploy ASP.Net Core, wil beginn **5th dec 2024 - complete 17 jan 2025** )

**Other changes:**
**Made a new migration,** possibly we have to fix / redo that?
csproj files in ChirpCore, -Infrastructure and Web was nessecary.
Deleted and created sln file, which actually is unnessecary now,
Cheep will not have a AuthorId stored as a int anymore, and nesecary changes other places in the code fx tests
updated tests folders, 
* not testing on id (which we dont have stored with our cheep anymore)
* csproj files updated w packages

**We have to update our api tests due to Program updates**

### CODE REVIEW

File: src/ChirpWeb/ChirpWeb.csproj [hunk 1]
```diff
@@ -11,34 +11,27 @@
...
+		<PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="8.0.10">
+    	<PrivateAssets>all</PrivateAssets>
+    	<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+  	</PackageReference>
+
+		<PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="8.0.7" />
     <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
+	</ItemGroup>
 
+	<ItemGroup>
```
Comment (Thyme-Cat) [inline src/ChirpWeb/ChirpWeb.csproj:30]: Skal der være noget i denne item group?

File: test/Chirp.ChirpWeb.Tests/API_Tests.cs
Comment (tomat-suppe) [inline test/Chirp.ChirpWeb.Tests/API_Tests.cs:1]: why is this class commented now?

File: src/ChirpWeb/Program.cs [hunk 2]
```diff
@@ -1,74 +1,105 @@
...
+			//miljøvariabel i kan være forskellige steder formateringer ifht forskellige terminaler (powershell, linux, mac osv) og computerer
+			//kig hvor jeres chirp.db, stifinder eller miljøvariabler på jeres computer
 
-var dbcon = new SqliteConnection(connectionString);
-//await dbcon.OpenAsync();
+		}
+		else
+		{*/
+			//This is for Global
+			connectionString = builder.Configuration["CHIRPDBPATH"] ?? throw new InvalidOperationException("Connectionstring not found locally, must be specified in another way maybe try: $env:CHIRPDBPATH=C:\tmp\ChirpData\chirp.db");
```
[the remark this answers is not in the document]
Comment (Thyme-Cat) [reply src/ChirpWeb/Program.cs:37]: I second this

[the remark this answers is not in the document]
Comment (tomat-suppe) [reply src/ChirpWeb/Program.cs]: Cat has pushed a new commit with this fixed. Branch can build now

**Comment by tomat-suppe** [comment]
This branch can't deploy

**Comment by tomat-suppe** [comment]
This branch was merged with ImplementingAuth0, and has been added to a new pull request to main

**Comment by tomat-suppe** [comment]
Edit: Not sure if this was completely merged into ImplementingAuth0. Reopening pull request

**Comment by tomat-suppe** [comment]
Re-closed. Have checked, everything is merged with implementingAuth0

**Pull request closed without merging**